src/App.tsx

```diff
@@ -1,85 +1,72 @@
-import './App.css'
-import {marked} from "marked";
-import DOMPurify from "dompurify";
-import {wrapAsync} from "./promise-wrapper";
+import React, { useState } from 'react';
+import { DownCircleOutlined, SettingOutlined } from '@ant-design/icons';
+import { useNavigate } from 'react-router-dom';
+import type { MenuProps } from 'antd';
+import { Menu } from 'antd';
+import NotFound from './components/notFound';
+import Nav from './components/Nav';
 
-export type AppProps = {
-  readonly contentServiceUrl: string;
-};
 
-export default function App({contentServiceUrl}: AppProps) {
-    const content = getContent(contentServiceUrl);
-    return (
-      <div>
-        <div dangerouslySetInnerHTML={{__html: renderMarkdown(content)}}/>
-      </div>
-    );
-}
-
-const getContent = wrapAsync('...loading',  async (baseUrl: string) => {
-  const cachedContent = getCachedContent();
-  if (cachedContent) {
-    console.log('using cached content');
-    return cachedContent;
-  }
-  const content = await getContentFromRestApi(baseUrl);
-  setCachedContent(content);
-  return content;
-});
-
-const getContentFromRestApi = async (baseUrl: string): Promise<string> => {
-  const now = Date.now();
-  const fiveDaysAgo = now - 1000 * 60 * 60 * 24 * 5;
-  const url = urlWithQueryParams(baseUrl, {
-    end: now.toString(),
-    start: fiveDaysAgo.toString()
-  });
-
-  const response = await fetch(url);
-  const [{content}] = await response.json();
-  console.log('using content from rest api' , content);
-
-  return content;
-};
 
-const urlWithQueryParams = (url: string, params: Record<string, string>) => {
-  const queryParams = Object.entries(params)
-    .map(([key, value]) => `${key}=${value}`)
-    .join('&');
-  return `${url}?${queryParams}`;
-};
 
-const renderMarkdown = (content: string) => {
-  return DOMPurify.sanitize(marked.parse(content));
-};
+const navigation = [
+  {
+    label: 'Home',
+    key: '/',
+    icon: <DownCircleOutlined />,
+
+  },
+  {
+    label: 'Link 1',
+    key: '/notfound1',
```

Comment:
Instead of having 3 different `notfound*`'s could we have one and just say `/notfound` for each?

Reply:
I think I tried that but it wanted all the keys to be unique.